mail/getmail6/Portfile

```
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        getmail6 getmail6 6.19.03 v
+github.tarball_from tarball
+
+revision            0
+categories          mail python
+platforms           {darwin any}
+maintainers         wolfsden.cz:~+macports
+license             GPL-2
+description         extensible mail retrieval system with POP3, IMAP4, SSL support
+long_description    getmail6 is a flexible, extensible mail retrieval          \
+                    system with support for POP3, IMAP4, SSL variants of both, \
+                    maildirs, mboxrd files, external MDAs, arbitrary message   \
+                    filtering, single-user and domain-mailboxes, and many      \
+                    other useful features.
+supported_archs     noarch
+
+conflicts           getmail
+
+homepage            https://getmail6.org
+
+checksums           rmd160  992edce4932d14b4fdd1ef84fa04c8040a6bad5d \
+                    sha256  b46a257bd221b7ee0e55f4f8e84e7a2fee8c05ec3b61c08298c3605df250a14a \
+                    size    215166
+
+patchfiles          0001-pop3-Support-use_xoauth2.patch
+patch.pre_args      -p1
+
+python.default_version  312
+
+post-destroot {
+    foreach mf { getmail getmail_fetch getmail_maildir getmail_mbox getmails } {
+        copy ${worksrcpath}/docs/${mf}.1 ${destroot}${prefix}/share/man/man1/
+    }
+
+    file mkdir ${destroot}${prefix}/share/doc/getmail/examples
+    copy ${worksrcpath}/docs/getmailrc-examples \
+        ${destroot}${prefix}/share/doc/getmail/examples/
+
+    foreach df { configuration.txt documentation.txt } {
+        copy ${worksrcpath}/docs/${df} ${destroot}${prefix}/share/doc/getmail/
+    }
```

You can do this without a `foreach` loop:

```suggestion
    xinstall -m 0644 -W ${worksrcpath}/docs configuration.txt documentation.txt ${destroot}${prefix}/share/doc/getmail/
```